.github/scripts/build-mdbook-summary.js

```diff
@@ -15,9 +15,15 @@ module.exports = async ({github, context}) => {
       if (!filename.endsWith(".md")) continue;
       const filePath = dirPath + filename
       const text = fs.readFileSync(filePath)
-      const title = text.toString().split(/\n/)
+      let title = text.toString().split(/\n/)
         .find(line => line.startsWith("# ") || line.startsWith(" # "))
         .replace("# ", "")
+      for (const markdownLink of title.matchAll(/\[(.*?)\]\((.*?)\)/g)) {
+        // Replace any [label](destination) markdown links in the title with just the label.
+        // This is because the titles are turned into links themselves,
+        // and we cannot have a link inside of a link - it breaks markdown and mdBook is not able to build.
```

Comment:
Wait, are you saing that having a link in the title of a MD is invalid MD?  
The we should rather fix the RFC document.

Reply:
No, not saying that, a link in the title is perfectly valid MD and we leave it untouched when rendering the document:

![Screenshot 2024-02-12 at 14 23 31](https://github.com/polkadot-fellows/RFCs/assets/12039224/2dc2be4e-2333-4201-97ee-1964b0f967f6)

This PR impacts only the left sidebar, because it consists of titles that are links for navigation in mdBook.

![Screenshot 2024-02-12 at 14 23 43](https://github.com/polkadot-fellows/RFCs/assets/12039224/2adb7103-e092-4a5a-ae9e-3d710f8e559a)

We can't have a link inside of a link in MD so this PR strips them, only in that place.

Reply:
Okay good.